src/main/webapp/index.html

```diff
@@ -2,15 +2,45 @@
 <html>
   <head>
     <meta charset="UTF-8" />
-    <meta name="description" content="Webapp That Helps You Choose A Place to Eat">
-    <meta name="keywords" content="restaurant, food, hungry, where, eat, random, help, lunch, dinner, indecisive">
+    <meta
+      name="description"
+      content="Webapp That Helps You Choose A Place to Eat"
+    />
+    <meta
+      name="keywords"
+      content="restaurant, food, hungry, where, eat, random, help, lunch, dinner, indecisive"
+    />
     <title>EatGoWhere</title>
     <link rel="stylesheet" href="style.css" />
-    <script defer src="script.js"></script>
+    <script src="script.js" defer></script>
+    <!-- Replace the value of the key parameter with your own API key. -->
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_HERE&libraries=places&callback=initMap"
+      defer
+    ></script>
   </head>
+
   <body>
     <h1>Welcome to EatGoWhere!</h1>
+    <p id="current-name"></p>
+    <p id="current-loc"></p>
     <p id="restaurant-name"></p>
-    <button onclick="getRandomRestaurant()">Submit</button>
+    <div class="pac-card" id="pac-card">
```

Comment:
What does pac stand for? Maybe something like searchbox-card will be better?

Reply:
This is following the places api css classes naming, https://developers.google.com/maps/documentation/javascript/places-autocomplete#places_searchbox

It should mean Places Auto Complete. What do you think?

Reply:
I see that's fine then!